File: Curso Logica de Programação Neri Neitzke/Algoritmos em Java/vetor_crescente.java
// Prof Neri Aldoir Neitzke   ULBRA
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
/*Problema: Crie um algoritmo lendo um vetor A de 15 elementos do tipo inteiro.
 Mostre esses elementos do vetor A. Reordene os elementos em ordem crescente,
 atualize o vetor A com os elementos na ordem crescente e mostre novamente o vetor A.*/
import javax.swing.JOptionPane;
class vetor_crescente
{
   public static void main(String args[])
   {
     int A[] = {9,3,7,6,21,50,45,36,2,8,7,9,42,32,18};
     int Aux;
     String mensagem = "Valores Originais\n";
     for(int j=0;j <= 14;j++)
          mensagem = mensagem+ "A["+j+"] ="+A[j]+" - ";

     for(int j=0;j <= 13;j++)
        for(int i=0;i <= 13;i++)
           if (A[i] > A[i+1])
           {
               Aux = A[i];
               A[i] = A[i+1];
               A[i+1] = Aux;
            };
     mensagem = mensagem + "\n\nValores Ordenados\n";
     for(int j=0;j <= 14;j++)
          mensagem = mensagem+ "A["+j+"] ="+A[j]+" - ";
    JOptionPane.showMessageDialog(null, mensagem);


   }
}

File: Curso Logica de Programação Neri Neitzke/Algoritmos em Pascal/pascal/vetor_crescente.pas
// Prof Neri Aldoir Neitzke         - ULBRA
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
{Problema: Crie um algoritmo lendo um vetor A de 3 elementos do tipo inteiro. Mostre esses elementos
do vetor A. Reordene os elementos em ordem crescente, atualize o vetor A com os elementos na ordem
crescente e mostre novamente o vetor A.}
program vetor_crescente;
var  A: array[1..3] of integer;
     i,j, aux : integer;
begin
     A[1] := 50;
     A[2] := 21;
     A[3] := 14;

     writeln('O valor de A[1] =',A[1]);
     writeln('O valor de A[2] =',A[2]);
     writeln('O valor de A[3] =',A[3]);
     for j := 1 to 2 do
        for i := 1 to 2 do
           if A[i] > A[i+1] then
           begin
              Aux := A[i];
              A[i]:= A[i+1];
              A[i+1] := aux;
           end;
     writeln;  //linha em branco
     writeln;  //linha em branco
     writeln('O valor de A[1] =',A[1]);
     writeln('O valor de A[2] =',A[2]);
     writeln('O valor de A[3] =',A[3]);

    writeln('..................Final do Sistema..................... ');
    readln();
end.
